site/en/_partials/devtools/whats-new.md

```diff
@@ -4,6 +4,22 @@ A list of everything that has been covered in the [What's New In DevTools](/tags
 
 {# $content #}
 
+### Chrome 115 {: #chrome115 }
+
+* [Elements improvements](/blog/new-in-devtools-115/#elements)
+* [New CSS subgrid badge](/blog/new-in-devtools-115/#subgrid)
```

Comment:
Can we remove this section? Subgrid actually isn't in Chrome yet so it's a bit odd to announce the DevTools functionality. Hang onto the copy and we'll use it when subgrid ships.